hazelcast/src/test/java/com/hazelcast/test/compatibility/SamplingNodeExtension.java

```diff
@@ -294,4 +296,10 @@ public CPPersistenceService getCPPersistenceService() {
     public JetService getJet() {
         throw new IllegalArgumentException();
     }
+
+    @Nullable
+    @Override
+    public JetServiceBackend getJetServiceBackend() {
+        return null;
```

Comment:
This looks wrong.

We can either delegate here or throw an exception with some meaningful message saying we don't run Jet tests to capture compatibility samples. Also if you could fix the same in the method above.

Reply:
I updated it and the method above.